bin/make-langpacks.py

```diff
@@ -0,0 +1,152 @@
+#!/usr/bin/env python
+"""
+This script builds up-to-date versions of Firefox Developer Edition (Aurora)
+language packs from mozilla source code and localizations and makes them
+available in a public repository.
+
+A `langpacks` folder is required on the same directory level as this script,
+with the following contents:
+* `mozilla-aurora`: SOURCE_REPOSITORY checkout
+* `mozilla-aurora/.mozconfig`: file set to reflect this folder structure, see
+  https://developer.mozilla.org/en-US/docs/Mozilla/Creating_a_language_pack#Pre-build_steps
+* `mozilla-aurora/langpacks/l10n`: contains L10N_REPOSITORIES checkouts
+* `mozilla-aurora/langpacks/build`: empty folder to contain build
+* `pontoon-langpacks`: TARGET_REPOSITORY checkout (you need write access)
+
+Required libraries:
+* lxml
+* requests
+"""
+
+
+from __future__ import print_function
+from lxml import html
+
+import datetime
+import os
+import requests
+import shutil
+import subprocess
+
+
+SOURCE_REPOSITORY = 'ssh://hg.mozilla.org/releases/mozilla-aurora/'
+TARGET_REPOSITORY = 'git@github.com:mathjazz/pontoon-langpacks.git'
+
+L10N_REPOSITORIES = [{
+    'url': 'ssh://hg.mozilla.org/releases/l10n/mozilla-aurora/{locale_code}/',
+    'locales_url': 'https://hg.mozilla.org/releases/l10n/mozilla-aurora/'
+}, {
+    'url': 'ssh://hg@bitbucket.org/mozilla-l10n/{locale_code}/',
+    'locales_url': 'https://api.bitbucket.org/2.0/repositories/mozilla-l10n'
+}]
+
+
+def write(text):
+    timestamp = datetime.datetime.now().strftime('[%Y-%m-%d %H:%M:%S] ')
+    print(timestamp + text)
+
+
+def execute(command, cwd=None):
```

Comment:
Thanks for bringing it up! It doesn't seem to include the error message, which we also need.